cumulus/pallets/executive/src/lib.rs

```diff
@@ -51,18 +51,48 @@ pub type OriginOf<E, C> = <CallOf<E, C> as Dispatchable>::Origin;
 // calculate the storage root outside the runtime after executing the extrinsic directly.
 #[frame_support::pallet]
 mod pallet {
-	use frame_support::pallet_prelude::*;
+	use frame_support::{
+		pallet_prelude::*, traits::UnfilteredDispatchable, weights::GetDispatchInfo,
+	};
 	use frame_system::pallet_prelude::*;
-	use sp_std::vec::Vec;
+	use sp_std::{boxed::Box, vec::Vec};
 
 	#[pallet::config]
-	pub trait Config: frame_system::Config {}
+	pub trait Config: frame_system::Config {
+		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
+		type Call: Parameter + UnfilteredDispatchable<Origin = Self::Origin> + GetDispatchInfo;
+	}
 
 	#[pallet::pallet]
 	#[pallet::generate_store(pub(super) trait Store)]
 	#[pallet::without_storage_info]
 	pub struct Pallet<T>(_);
 
+	#[pallet::call]
+	impl<T: Config> Pallet<T> {
+		// TODO: this call may be moved to other places if this pallet is no longer necessary.
+		/// Unsigned version of `frame_sudo::sudo_unchecked_weight`.
+		#[allow(clippy::boxed_local)]
+		#[pallet::weight((*_weight, call.get_dispatch_info().class, Pays::No))]
+		pub fn sudo_unchecked_weight_unsigned(
+			origin: OriginFor<T>,
+			call: Box<<T as Config>::Call>,
+			_weight: Weight,
```

Comment:
Should it be `_weight` then (if it is actually used above)?

Reply:
`*_weight` actually a dereference to `_weight` as `ref _weight` will be used when using the declared weight.